apps/official/index.ts

```diff
@@ -133,6 +157,7 @@ export const config: AppConfig = {
         {
           id: "areas-fill",
           source: "CI-to-plz",
+          type: LayerType.FILL,
```

Comment:
Kann man das darunter in der `fn` dann raus nehmen? Sieht gedoppelt aus.

Reply:
glaub nicht er erstellt ja nochmal ein nested object.
Wäre aber auch dafür einen eine clickable boolean zu setzen oder dadrunter ein clickable layers array.
halt die frage ob es auch in die "visuals" soll oder irgendwo einzeln